### PR TITLE
Problem: recent bug fixes on ethermint not used

### PR DESCRIPTION
**Pull request by yihuang**
- [Quick fix for eth_feeHistory when reward is nil](https://github.com/tharsis/ethermint/pull/975)
- [Add buffer to eth_gasPrice response](https://github.com/tharsis/ethermint/pull/968)
- [Fix GasMeter reset in AnteHandler, aka. return correct gasWanted to tendermint](https://github.com/tharsis/ethermint/pull/964)
  modified to only fix checkTx mode to avoid breaking consensus.
- [add returnValue message on tracing](https://github.com/tharsis/ethermint/pull/962)

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

**Comment by CeruleanAtMonaco** [comment]
what about https://github.com/tharsis/ethermint/pull/955?

**Comment by yihuang** [comment]
> what about [tharsis/ethermint#955](https://github.com/tharsis/ethermint/pull/955)?

it's already included by this PR: https://github.com/crypto-org-chain/cronos/pull/370

File: go.mod [hunk 1]
```diff
@@ -164,7 +164,7 @@ replace (
 	// See https://github.com/tecbot/gorocksdb/pull/216
 	github.com/tecbot/gorocksdb => github.com/cosmos/gorocksdb v1.1.1
 
-	github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.10.0-alpha1-cronos-6.0.20220301081024-4ec8b052c769
+	github.com/tharsis/ethermint => github.com/crypto-org-chain/ethermint v0.10.0-alpha1-cronos-7
```
Comment (yihuang) [inline go.mod:167]: https://github.com/crypto-org-chain/ethermint/compare/4ec8b052c769...v0.10.0-alpha1-cronos-7?expand=1

**Review by thomas-nguy — APPROVED** [review]
looks okay 👍

**Pull request merged**